src/components/autosuggest/_macro-options.md

```diff
@@ -18,4 +18,4 @@
 | input               | `Input` [_(ref)_](/components/input) | true     | Configuration object for the input                                                                                                                                           |
 | language            | string                               | false    | The ISO 639-1 Code will override the default language in page. Please note that only 'en', 'cy' and 'ni' is currently supported                                              |
 | resultsThreshold    | float                                | false    | Option to adjust the search threshold and fuzziness. Accepts a range from 0 to 1, where 0 provides the closest match and 1 allows for more distant matches. Defaults to 0.2. |
-| id                  | string                               | false    | The `id` of the input                                                                                                                                                        |
+| id                  | string                               | true     | The `id` of the input                                                                                                                                                        |
```

Comment:
I assume this is a correction rather than an alteration since I can't see a change in the code to require this field and there is no mention in the ticket itself?

Reply:
I think this has been updated as a correction since technically this is required given it sets the id of the input in the macro and that param is required on the input component